docs/guide.md

```diff
@@ -80,33 +82,54 @@ Make sure to check the [**Compatibility with other Fabric mods**](#compatibility
    Click OK and close the installer.
 8. Download Iris and Sodium from your website of choice:
     - Iris Shaders: [Curseforge](https://www.curseforge.com/minecraft/mc-mods/irisshaders), [Modrinth](https://modrinth.com/mod/iris)  
-    - Sodium: [Curseforge](https://www.curseforge.com/minecraft/mc-mods/sodium), [Modrinth](https://modrinth.com/mod/sodium)
+    - Sodium: [Curseforge](https://www.curseforge.com/minecraft/mc-mods/sodium), [Modrinth](https://modrinth.com/mod/sodium)  
+   
+   The following table shows which version of Sodium you need based on your Minecraft version:
+   
+   | Minecraft    | 1.16.5       | 1.17.1       | 1.18.1       |
```

Comment:
I'd recommend transposing this table, since it could get very wide.

Reply:
Done